hooks/git_commit_standards_hook.py

```
@@ -91,23 +123,51 @@ def main():
                         }))
                         return 1
     
-    # Check for git push without protection
-    if re.match(r'^git\s+push\s+', command):
-        # Check if pushing to main/master directly
-        if re.search(r'(main|master|production)\b', command):
+    # Check for git push to protected branches
+    if re.search(r'git\s+push', command):
+        # Check if pushing to main/master/production directly
+        if re.search(r'origin\s+(main|master|production)\b', command) or \
+           re.search(r'(main|master|production):(main|master|production)', command):
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)